flask_admin/static/admin/js/form-1.0.0.js

```diff
@@ -155,7 +155,7 @@
         }
 
         // set up tiles
-        L.tileLayer('http://{s}.tiles.mapbox.com/v3/'+MAPBOX_MAP_ID+'/{z}/{x}/{y}.png', {
```

Comment:
Has v3 API completely stopped functioning? Or will these changes break existing functioning code running on the v3 API?